app/errors.py

```diff
@@ -0,0 +1,17 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotWearingMaskError(Exception):
+    def __str__(self) -> str:
+        return "NotWearingMaskError()"
+
+
+class NotVaccinatedError(VaccineError):
+    def __str__(self) -> str:
+        return "All friends should be vaccinated"
+
+
+class OutdatedVaccineError(VaccineError):
+    def __str__(self) -> str:
+        return "All friends should be vaccinated"

```

Comment:
![image](https://user-images.githubusercontent.com/103201010/220103212-c05aadb9-6e82-42cc-a274-967ea832b275.png)
it is not descriptive messages for exceptions, fix this, please